app/controllers/api/prediction_judgements_controller.rb

```diff
@@ -0,0 +1,10 @@
+module Api
+  class PredictionJudgementsController < AuthorisedController
+    def create
+      @prediction = Prediction.find(params[:prediction_id])
+      raise UnauthorizedRequest unless @user.authorized_for(@prediction)
+      @prediction.judge!(params[:outcome], @user)
+      render status: :created, json: nil
```

Comment:
Since the endpoint is adding a row to the judgement model and not modifying the prediction I thought it more appropriate to return the JSONified judgement.